src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -933,10 +933,7 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <comment>{0} is a version number</comment>
   </data>
   <data name="Warning_HTTPSource" xml:space="preserve">
-    <value>Warning: Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS' source.</value>
-  </data>
-  <data name="Warning_NewHTTPSource_VSOptions" xml:space="preserve">
-    <value>Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS' source.</value>
+    <value>`AllowInsecureConnections` attribute has been enabled for this source which would allow it connecting to a NON-HTTPS source.</value>
```

Comment:
@JonDouglas  any suggestions on this. It is a warning shown to users when the `allowInsecureConnections` attribute is set to true for a source. See video.

Reply:
`AllowInsecureConnections` is enabled and allows http access. Note: This method is not secure. For secure options, see https://aka.ms/nuget-https-everywhere.